source/state_representation/include/state_representation/State.hpp

```diff
@@ -90,18 +90,12 @@ class State : public std::enable_shared_from_this<State> {
   void set_filled();
 
   /**
-   * @brief Getter of the timestamp attribute
+   * @brief Getter of the time since last modification attribute
    */
-  const std::chrono::time_point<std::chrono::steady_clock>& get_timestamp() const;
+  const std::chrono::time_point<std::chrono::steady_clock>& get_epoch() const;
```

Comment:
I think you can leave `get_timestamp` as a utility function the way it was, and then add a much nicer utility method to get the age in seconds

```c++
  /**
   * @brief Getter of the time since last modification in seconds
   */
  double get_age() const;
```

If we use seconds for convenience, you can get the age as follows: 
```c++
  std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::steady_clock::now() - this->timestamp_).count() * 1e9
```